docs/modules/ROOT/partials/getting-started/con-registry-serdes-strategy.adoc

```diff
@@ -4,18 +4,30 @@
 [id='registry-serdes-concepts-strategy-{context}']
 = Strategies to lookup a schema
 
-The Kafka client serializer/deserializer uses a lookup _strategy_ to determine the artifact ID or global ID under which the message schema is registered in {registry}.
+The Kafka client serializer uses two lookup _strategies_ to determine the artifact ID and global ID under which the message schema is registered in {registry}.
 
-For a given topic and message, you can use implementations of the following Java classes:
+For a given topic and message, you can use implementations of the following Java interfaces:
 
 * `ArtifactIdStrategy` to return an artifact ID
 * `GlobalIdStrategy` to return a global ID
 
-The artifact ID returned depends on whether the _key_ or _value_ in the message is being serialized.
```

Comment:
Suggest adding block heading to help break up the text, for example: ".Artifact ID strategy"